Makefile

```diff
@@ -23,6 +23,10 @@ DOCKER?=$(if $(or $(in_docker_group),$(is_root)),docker,sudo docker)
 
 .DEFAULT_GOAL := all
 
+CUR_DIR = $(shell pwd)
+
+include ./Makefile.docker
```

Comment:
Do you think splitting current `Makefile` would help?

I can think of slicing it in the following way

- build
- deployment
- docker-plugins
- docker-builder

If we do it how about moving all those files to `.make` directory?

Reply:
If so we should add it to our `test-keeper.yml` config.

Reply:
sounds good. how about these names:
`Makefile.build`
`Makefile.deploy`
`Makefile.docker.binary` or `Makefile.docker.prow` or just `Makefile.docker.plugins`
`Makefile.docker.build`

Reply:
Opting for `Makefile.docker.prow`